apple/MarkdownFormatter.h

```diff
@@ -6,6 +6,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 const NSAttributedStringKey RCTLiveMarkdownTextAttributeName = @"RCTLiveMarkdownText";
 
+const NSAttributedStringKey RCTLiveMarkdownTextBackgroundAttributeName = @"RCTLiveMarkdownTextBackground";
+@interface RCTMarkdownTextBackground : NSObject
+  @property (nonatomic, strong) UIColor *color;
+  @property (nonatomic, assign) CGFloat cornerRadius;
+@end
```

Comment:
I'm not sure if this is the right place to put this object in
@tomekzaw do you have a better idea? Maybe a separate file for RCTMarkdownTextBackground and RCTMarkdownTextBackgroundWithRange? 🤔

Reply:
We can add that in a separate file